phpstan.neon

```diff
@@ -273,6 +273,8 @@ parameters:
                  # traversing is complex operatoin
                  - packages/simple-php-doc-parser/src/PhpDocNodeTraverser.php
                  - packages/php-config-printer/src/NodeFactory/ArgsNodeFactory.php
+                 # resolving node value is complex operation
+                 - packages/astral/src/NodeValue/NodeValueResolver.php
```

Comment:
`FuncCallValueResolver` service should be extracted instead of ignoring. It's already quite huge class to manage and asks for decoupling

Reply:
Yes. Name resolver in Rectot is split in this design. One more reason to refactor this to collector, before it.becomes Gordic knot :)

Reply:
I'm new to this package :) could you refactor it please?

Reply:
The best way to learn is start :) one change a time is enough. The `SimpleNameResolverStaticFactory` design approach is ok.

I would, but I have focus elsewhere now.

Reply:
To make a bit easier for you :)
https://github.com/symplify/symplify/pull/3628

Reply:
thanks ;) I moved it to separate class